examples/dt/Adoption/DFG-upgrades-adoption-18.0-from-17.1-passed-phase2-rhev-3cont-2comp-ipv4-HA-geneve-ovn/DFG-upgrades-adoption-18.0-from-17.1-passed-phase2-rhev-3cont-2comp-ipv4-HA-geneve-ovn.md

```diff
@@ -0,0 +1,55 @@
+# Deployed Topology DFG-upgrades-adoption-18.0-from-17.1-passed-phase2-rhev-3cont-2comp-ipv4-HA-geneve-ovn
+
+## General information
+
+| Revision | Change                | Date             |
+|--------: | :-------------------- | :--------------: |
+| v0.1     | Initial publication   | 22.12.23      |
+
+## Node topology
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Openshift master/worker combo-node cluster    | vm    | 3      |
+| Undercloud node - Source osp 17               | vm    | 1      |
+| Controller nodes - Source osp 17              | vm    | 3      |
+| Compute nodes    - Souce osp 17               | vm    | 2      |
+
+
+## Services, enabled features and configurations
+| Service                                     | configuration                   | Lock-in coverage?  |
+| ------------------------------------------- | ------------------------------- | ------------------ |
+| RabbitMQ                                    | default                         | Must have          |
+| OVN                                         | default                         | Must have          |
+| galera                                      | default                         | Must have          |
+| Keystone                                    | default                         | Must have          |
+| placement                                   | default                         | Must have          |
+| nova                                        | default                         | Must have          |
+| neutron                                     | default                         | Must have          |
```

Comment:
We're also using Glance to boot a VM but i'm not sure how exhaustive this list has to be. LGTM